tutorials/elixir-phoenix-on-google-app-engine.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Run an Elixir Phoenix app on the Google App Engine Flexible Environment
+description: Learn how to deploy a Phoenix app to the Google App Engine flexible environment.
+author: dazuma
+tags: App Engine, Elixir, Phoenix
+date_published: 2017-10-11
+---
+
+The [Google App Engine flexible environment](https://cloud.google.com/appengine/docs/flexible/)
+is an easy way to deploy your apps to the same infrastructure that powers
+Google's products. Using the open source
+[Elixir Runtime](https://github.com/GoogleCloudPlatform/elixir-runtime), your
+app written in [Elixir](http://elixir-lang.org/) with the
+[Phoenix](http://phoenixframework.org/) Framework can be up and running in App
+Engine in minutes.
+
+This tutorial will help you get started deploying a Phoenix app to App Engine.
+You will create a new Phoenix application, and learn how to configure, deploy,
+and update it. For simplicity, this tutorial app will not use Ecto or connect
+to a SQL database, but you can extend it to connect to Google Cloud SQL or any
+other database service.
+
+This tutorial requires Elixir 1.4 and Phoenix 1.3 or later. It assumes you are
+already familiar with basic Phoenix web development.
+
+## Before you begin
+
+Before running this tutorial, take the following steps:
+
+1.  Use the [Google Cloud Platform Console](https://console.cloud.google.com/)
+    to create a new Cloud Platform project.
+
+2.  Enable billing for your project.
+
+3.  Install the [Google Cloud SDK](https://cloud.google.com/sdk/). Make sure
+    you [initialize](https://cloud.google.com/sdk/docs/initializing) the SDK
+    and set the default project to the new project you created.
+
+    Version 175.0.0 or later of the SDK is required.
+
+If you have not yet installed Elixir and Phoenix, do so:
```

Comment:
I'm not convinced it's necessary to separate them. I already stated further above that the tutorial assumes you're already familiar with using Phoenix, which suggests you already have it installed or know how to do so. This section was mostly for completeness. If we separate them, we'd probably need to do things like justify why we're installing node (because Phoenix uses brunch by default to manage assets) etc., which is extraneous information for this tutorial.